### DISCUSSION TITLE
Width Height Fit Content

### DISCUSSION THREAD

**Discussion by Milkhan** [nextapps-de/winbox#131] [Ideas]
Thank you for this gorgeous package,

Is it possible to remove the size of the window from style so we can add fit content through css?

Currently, when we add a video player or any text, it has some issues regarding width and height, it doesn't calculate content's width and height, we have to define it by hand. Previously we were using our own modal and we could add CSS attributes for fit content. But it's impossible with this package because width and height are setting programmatically with the style attributes.

Any suggestions?

Thank you

**Comment by ts-thomas** [comment]
Hello, please try to adding `!important` on your CSS attribute, this should work.

**Comment by kodie** [comment]
Ideally, if a `width` or `height` isn't set, it should just be the size of the content.

**Comment by ts-thomas** [comment]
The option "autosize" is now supported.
